ts/src/renderer/phaser/scenes/GameScene.ts

```diff
@@ -209,7 +209,7 @@ class GameScene extends PhaserScene {
 
 	private setZoomSize (height: number): void {
 		// backward compatible game scaling on average 16:9 screen
```

Comment:
comment no longer applies

Reply:
comments still applies, maybe even more. we might not be using numbers 16 and 9 explicitly but multiplying with chosen value actually accomplishes backward compatibility more accurately on 16:9 screens